### PR TITLE
Fix registration-confirmed test, remove 'lets' from event titles

### PR DESCRIPTION
**Pull request by dannypernik**
#529 WIP

### CODE REVIEW

**Comment by nickolashe** [comment]
@dannypernik what exactly does this need from the backend? also, could you resolve the conflicts?

**Comment by nickolashe** [comment]
if you pull develop the tests should be green

**Comment by dannypernik** [comment]
registration-confirmed template needs `{% if coordinator %}`. I tried copying things over from EventDetailView but couldn't get it working.

File: openCurrents/views.py [hunk 1]
```diff
@@ -2351,11 +2351,17 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class RegistrationConfirmedView(DetailView, LoginRequiredMixin):
+class RegistrationConfirmedView(LoginRequiredMixin, SessionContextView, DetailView):
```
Comment (nickolashe) [inline openCurrents/views.py]: @dannypernik note that the order of parent views is important

File: openCurrents/templates/invite-volunteers.html [hunk 2]
```diff
@@ -110,7 +110,57 @@ <h3 class="title-sub">
 
 {% block popups %}
   <div id="preview-email-popup" class="modal center small-12 medium-9 large-7 small-centered columns">
+<<<<<<< HEAD
```
Comment (nickolashe) [inline openCurrents/templates/invite-volunteers.html]: @dannypernik this will require resolving the conflict

Comment (dannypernik) [reply openCurrents/templates/invite-volunteers.html]: @nickolashe oops. fixed.

File: openCurrents/views.py (hunk 1 above)
Comment (dannypernik) [reply openCurrents/views.py]: Ah interesting. Noted, thanks

**Comment by dannypernik** [comment]
@nickolashe I believe this is ready to merge into develop and master

**Comment by nickolashe** [comment]
@dannypernik test failure is not related to this PR, merging

**Comment by nickolashe** [comment]
@tevtonez @dannypernik let's try to not get in the habit of ignoring test failures in develop

**Pull request merged**